imgui-SFML.cpp

```diff
@@ -686,29 +688,85 @@ void Image(const sf::RenderTexture& texture, const sf::Vector2f& size, const sf:
 /////////////// Image Overloads for sf::Sprite
 
 void Image(const sf::Sprite& sprite, const sf::Color& tintColor, const sf::Color& borderColor) {
-    sf::FloatRect bounds = sprite.getGlobalBounds();
-    Image(sprite, sf::Vector2f(bounds.width, bounds.height), tintColor, borderColor);
+	Image(sprite, sprite.getGlobalBounds().getSize(), sf::Transform::Identity, tintColor, borderColor);
 }
 
 void Image(const sf::Sprite& sprite, const sf::Vector2f& size, const sf::Color& tintColor,
-           const sf::Color& borderColor) {
-    const sf::Texture* texturePtr = sprite.getTexture();
-    // sprite without texture cannot be drawn
-    if (!texturePtr) {
+    const sf::Color& borderColor) {
+	Image(sprite, size, sf::Transform::Identity, tintColor, borderColor);
+}
+
+void Image(const sf::Sprite& sprite, const sf::Vector2f& size, const sf::Transform& additionalTransform,
+    const sf::Color& tintColor, const sf::Color& borderColor)
+{
+    const sf::IntRect& textureRect = sprite.getTextureRect();
+
+    // sprite without texture on empty frame cannot be drawn
+    if (!sprite.getTexture() ||
```

Comment:
This check should be done before getting textureRect

Reply:
Not check necessary, as the texture rect is a value member of ``sf::Sprite``: https://github.com/SFML/SFML/blob/master/include/SFML/Graphics/Sprite.hpp